pkg/database/sqlserver.go

```diff
@@ -0,0 +1,208 @@
+package database
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/fraenky8/tables-to-go/pkg/settings"
+
+	// sqlserver database driver
+	_ "github.com/denisenkom/go-mssqldb"
+)
+
+// Sqlserver implements the Database interface with help of GeneralDatabase.
+type Sqlserver struct {
+	*GeneralDatabase
+
+	defaultUserName string
+}
+
+// NewSqlserver creates a new Sqlserver database.
+func NewSqlserver(s *settings.Settings) *Sqlserver {
+	return &Sqlserver{
+		GeneralDatabase: &GeneralDatabase{
+			Settings: s,
+			driver:   dbTypeToDriverMap[s.DbType],
+		},
+		defaultUserName: "sa",
+	}
+}
+
+// Connect connects to the database by the given data source name (dsn) of the
+// concrete database.
+func (ss *Sqlserver) Connect() error {
+	return ss.GeneralDatabase.Connect(ss.DSN())
+}
+
+// DSN creates the DSN String to connect to this database.
+func (ss *Sqlserver) DSN() string {
```

Comment:
Lets add a unit test for that? I see the other database implementation have one, just for the sake of consistency?